cosmos-cmd-tlm-api/app/controllers/api_controller.rb

```diff
@@ -69,7 +69,8 @@ def api
       request_headers = Hash[*request.env.select {|k,v| k.start_with? 'HTTP_'}.sort.flatten]
       request_data = req.body.read
       status, content_type, body = handle_post(request_data, request_headers)
-      Cosmos::Logger.info("User performed API action: #{request_data} #{request_headers}", scope: params[:scope], user: user_info(request.headers['HTTP_AUTHORIZATION']))
+      Cosmos::Logger.info("API data: #{request_data}", scope: params[:scope], user: user_info(request.headers['HTTP_AUTHORIZATION']))
+      Cosmos::Logger.debug("API headers: #{request_headers}", scope: params[:scope], user: user_info(request.headers['HTTP_AUTHORIZATION']))
```

Comment:
headers include secrets, so need to make sure debug logging is not on normally, or sanitize the secrets before logging.

Reply:
Info is higher than debug so normally you don't see debug. This is better than before where it was all coming out with info.